languages/python/sqlalchemy-oso/requirements-dev.txt

```diff
@@ -2,4 +2,4 @@ black==21.5b0
 flake8==3.9.2
 mypy==0.812
 sqlalchemy-stubs==0.4
-tox==3.23.1
+tox==3.23.1
```

Comment:
what in the heck changed here? I'm stumped

Reply:
lol